distance-calculator/calculator/src/main/java/io/pravega/flinkapp/InfluxdbSink.java

```diff
@@ -17,13 +19,12 @@ public class InfluxdbSink extends RichSinkFunction<OutSenorData> {
     String influxdbPassword = "";
     String influxdbDbName = "";
 
-    public InfluxdbSink() {}
 
-    public InfluxdbSink(String influxdbUrl, String influxdbUsername, String influxdbPassword, String influxdbDbName) {
-        this.influxdbUrl = influxdbUrl;
-        this.influxdbUsername = influxdbUsername;
-        this.influxdbPassword = influxdbPassword;
-        this.influxdbDbName = influxdbDbName;
+    public InfluxdbSink() {
+        this.influxdbUrl = "http://project-metrics:8086";
```

Comment:
Can we keep these hard-coded values in the properties file and read them from there

Reply:
they are not hardcoded values , they are defaults and will be overwritten when we fetch information from the flink prop files. Line 51-57 does exactly that .